### PR TITLE
Added fastai example

### PR DESCRIPTION
**Pull request by mattmcclean**
Added a new advanced example showing how to train and deploy a fastai based model using Amazon SageMaker. Uses the PyTorch container which now contains the fastai library.

### CODE REVIEW

**Comment by mattmcclean** [comment]
Yes, it has been tested with a SageMaker notebook

File: advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a training job using the sagemaker.PyTorch estimator\n",
+    "\n",
+    "The `PyTorch` class allows us to run our training function on SageMaker. We need to configure it with our training script, an IAM role, the number of training instances, and the training instance type. \n",
+    "\n",
+    "For local training with GPU, we could set this to ```local_gpu```.  In this case, `instance_type` was set above based on your whether you're running a GPU instance. If `instance_type` is set to a SageMaker instance type (e.g. ml.p2.xlarge) then the training will happen on SageMaker.\n",
```
Comment (neelamgehlot) [inline advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb]: [nit] In this case, `instance_type` was set ~above~ *below* based on ~your~ whether you're running a GPU instance

File: advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+    "pets_estimator.delete_endpoint()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```
Comment (neelamgehlot) [inline advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb]: [nit] Empty Cell

File: advanced_functionality/fastai_oxford_pets/setup.sh [hunk 3]
```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
```
[the remark this answers is not in the document]
Comment (mattmcclean) [reply advanced_functionality/fastai_oxford_pets/setup.sh]: I have removed the script but kept the example `daemon.json` file as the default docker setting for `default-shm-size` is too low when training locally with docker-compose. In the notebook it copies the new `daemon.json` and restarts the Docker daemon.

File: advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "fastai"
```
[the remark this answers is not in the document]
Comment (mattmcclean) [reply advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb]: I have changed the kernel name to `conda_pytorch_p36`

File: advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb (hunk 1 above)
Comment (mattmcclean) [reply advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb]: Updated docs in notebook

File: advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb (hunk 2 above)
Comment (mattmcclean) [reply advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb]: removed empty cell

**Comment by mattmcclean** [comment]
The example notebook is different than the notebooks from https://github.com/fastai/course-v3/tree/master/nbs/dl1 as it uses SageMaker for training and hosting rather than doing it all on the notebook instance.

**Pull request merged**